docs/configuration/index.md

```diff
@@ -25,6 +25,7 @@ Some configuration parameters are primarily controlled via NetBox's admin interf
 * [`CUSTOM_VALIDATORS`](./data-validation.md#custom_validators)
 * [`DEFAULT_USER_PREFERENCES`](./default-values.md#default_user_preferences)
 * [`ENFORCE_GLOBAL_UNIQUE`](./miscellaneous.md#enforce_global_unique)
+* [`EVENTS_PIPELINE`](./miscellaneous.md#events_pipeline)
```

Comment:
This isn't a [dynamic config parameter](https://github.com/netbox-community/netbox/blob/develop/netbox/netbox/config/parameters.py) (one which can be modified via the UI).
```suggestion
```